doc/index.qdoc

```diff
@@ -0,0 +1,79 @@
+/****************************************************************************************
+**
+** Copyright (C) 2021 Open Mobile Platform LLC
+** All rights reserved.
+**
+****************************************************************************************/
+
+/*!
+\page index.html
+\title Sailfish OS WebView Component
+
+The Sailfish OS WebView Component provides a straightforward and flexible way
+for you to introduce access to Web pages and websites into your applciation.
+
+The WebView component utilises the Sailfish OS browser engine, built using
+Mozilla gecko.
+
+\section1 Using the Sailfish OS WebView Component
+
+Currently you have to explicitly link against qtmozembed-qt5 at build time, to
+avoid a symbol issue which leads to a crash:
+
+\code
+CONFIG += link_pkgconfig
+PKGCONFIG += qt5embedwidget
+\endcode
+
+\section2 C++ API
+
+The C++ API consists of the following classes:
+
+\list
+\li the \l{SailfishOS}{WebEngine} class
```

Comment:
I think these two links might be incorrectly formatted, although I might be wrong

Reply:
I've amended this.